src/Sarif.UnitTests/SarifCurrentToVersionOneTests.cs

```diff
@@ -0,0 +1,202 @@
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""results"": []
+    }
+  ]
+}";
+
+            VerifyCurrentToVersionOneTransformation(v2LogText, v1LogExpectedText);
+        }
+
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_MinimumWithTwoRuns()
+        {
+            string v2LogText =
+              @"{
+                  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+                  ""version"": ""2.0.0"",
+                  ""runs"": [
+                    {
+                      ""tool"": {
+                        ""name"": ""CodeScanner"",
+                        ""semanticVersion"": ""2.1.0""
+                      },
+                      ""results"": []
+                    },
+                    {
+                      ""tool"": {
+                        ""name"": ""AssetScanner"",
+                        ""semanticVersion"": ""1.7.2""
+                      },
+                      ""results"": []
+                    }
+                  ]
+                }";
+
+            string v1LogExpectedText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-1.0.0"",
+  ""version"": ""1.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner"",
+        ""semanticVersion"": ""2.1.0""
+      },
+      ""results"": []
+    },
+    {
+      ""tool"": {
+        ""name"": ""AssetScanner"",
+        ""semanticVersion"": ""1.7.2""
+      },
+      ""results"": []
+    }
+  ]
+}";
+
+            VerifyCurrentToVersionOneTransformation(v2LogText, v1LogExpectedText);
+        }
+
+        [Fact]
+        public void SarifTransformerTests_ToVersionOne_OneRunWithLogicalLocations()
+        {
+            string v2LogText =
+              @"{
+                  ""version"": ""2.0.0"",
+                  ""runs"": [
+                    {
+                      ""tool"": {
+                        ""name"": ""CodeScanner"",
+                        ""semanticVersion"": ""2.1.0""
+                      },
+                      ""logicalLocations"": {
+                        ""collections::list::add"": {
+                          ""name"": ""add"",
+                          ""kind"": ""function"",
+                          ""parentKey"": ""collections::list""
+                        },
+                        ""collections::list"": {
+                          ""name"": ""list"",
+                          ""kind"": ""type"",
+                          ""parentKey"": ""collections""
+                        },
+                        ""collections"": {
+                          ""name"": ""collections"",
+                          ""kind"": ""namespace""
+                        }
+                      },
+                      ""results"": [
+                      ]
+                    }
+                  ]
+                }";
+
+            string v1LogExpectedText =
```

Comment:
As a matter of course, I `const` strings that aren't expected to change (and then follow the capitalization convention for `const`):
```
const string V2LogText = @"...";
```

Reply:
It's just one more way the compiler can help you out a little, at almost no cost to you.